pkg/prometheus/common.go

```diff
@@ -474,3 +475,20 @@ func BuildStatefulSetService(name string, selector map[string]string, p monitori
 
 	return svc
 }
+
+func ConvertLabelSelectorRequirementToRequirementSelector(requirement metav1.LabelSelectorRequirement) (*selection.Operator, error) {
```

Comment:
(suggestion) maybe we can pass requirement.Operator directly? we can also return a value rather than a pointer?
```suggestion
func ConvertRequirementOperator(operator metav1.LabelSelectorOperator) (selection.Operator, error) {
```